css/css.md

```diff
@@ -38,6 +38,8 @@ About 4.5 hours
 - Describe the hierarchy of specificity for CSS selectors
 - Describe the different parts of the "Box" model
 - Effectively use the `display` property
+- Changing styles from CSS using `Chrome Dev Tools`
```

Comment:
can you state this in one line please? ie "Changing and removing..."

Reply:
also, can you format Chrome Dev Tools in another way?  This way denotes code or commands.

Reply:
still waiting on some small changes @pydevsg

Reply:
I will make the changes as soon as possible @alodahl .

Reply:
Did you change this one?